ethdb/database.go

```diff
@@ -150,6 +154,7 @@ func (db *LDBDatabase) Close() {
 	}
 }
 
+// LDB returns the memory address for leveldb.DB.
```

Comment:
`// LDB returns the internal leveldb.DB instance.`

Reply:
Thank you for all the additions, corrections, and feedback on this module, Peter. I'll be updating the PR tomorrow.